infra/aws/e2e/accounts/tf/README.md

```diff
@@ -0,0 +1,35 @@
+# AWS account scripting
+
+* We create AWS accounts using terraform.
+
+* The state file is stored (encrypted) in GCS.
+
+* The encryption key is available only to those that should be running
+  this, as it contains the AWS account keys.
+
+* We additionally generate the boskos objects, and upload that also to
+  GCS, encrypted using the same key (as it broadly has the same data)
+
+
+## Applying to terraform
+
+Ensure that the key is written to `gcs-encryption-key`
```

Comment:
Who has access to this?

Ideally this key would be stored in google cloud secret manager, and access would be restricted to a @kubernetes.io group we manage.  e.g.

```sh
KEY=$(gcloud secrets --project=kubernetes-public versions access latest --secret=gcs-encryption-key)
```